keras_nlp/models/gpt2.py

```diff
@@ -25,6 +29,108 @@ def _gpt_2_kernel_initializer(stddev=0.02):
     return keras.initializers.RandomNormal(stddev=stddev)
```

Comment:
I noticed the following warning in your notebook. Is this a problem? cc @qlzh727 
```
/usr/local/lib/python3.7/dist-packages/keras/initializers/initializers_v2.py:121: UserWarning: The initializer RandomNormal is unseeded and being called multiple times, which will return identical values  each time (even if the initializer is unseeded). Please update your code to provide a seed to the initializer, or avoid using the same initalizer instance more than once.
  f"The initializer {self.__class__.__name__} is unseeded "
```

Definitely did not see this for `keras_nlp.models.BertBase` and @qlzh727 says this should work. Could the notebook error be old?

Reply:
Oh! This is actually a legit bug! https://github.com/keras-team/keras-nlp/issues/352

I actually have a branch knocking this out, just haven't had time to put it up. Will do shortly

Reply:
The tl;dr is we might be seeing this from error in our layers themselves, not the model code.

Reply:
https://github.com/keras-team/keras-nlp/pull/371

Reply:
OK, in that case we just need to reconcile with #361

Reply:
I have manually removed this error message from the notebooks.

Reply:
Should now be fixed at master! But we should keep an eye out for these errors. We do need to fix them.